src/config_resolver.go

```diff
@@ -16,10 +16,11 @@ package main
 import (
 	"bytes"
 	"encoding/json"
-	"github.com/elodina/go-avro"
 	"os"
 	"text/template"
 	"time"
+
+	"github.com/elodina/go-avro"
```

Comment:
It's the formatting in vscode that splits the built-in packages from the rest I think.